yarpl/include/yarpl/flowable/EmitterFlowable.h

```diff
@@ -36,17 +36,9 @@ class EmiterSubscription : private Subscription, private Subscriber<T> {
       Reference <EmiterBase<T>> emiter,
       Reference <Subscriber<T>> subscriber)
       : emiter_(std::move(emiter)), subscriber_(std::move(subscriber)) {
-    // We expect to be heap-allocated; until this subscription finishes
-    // (is canceled; completes; error's out), hold a reference so we are
-    // not deallocated (by the subscriber).
-    Refcounted::incRef(*this);
     subscriber_->onSubscribe(get_ref<Subscription>(this));
   }
 
-  virtual ~EmiterSubscription() {
-    subscriber_.reset();
```

Comment:
removed as this is redundant code; subscriber_ is reset when ~Reference is called